### ISSUE TITLE
Display LLOQ in time profiles

### ISSUE THREAD

**Issue by Felixmil** [Open-Systems-Pharmacology/TLF-Library#422]

**Comment by Yuri05** [comment]
"Very few new code lines" for the option 1 is is a "pro" and not a "con" I guess 😉

**Comment by Felixmil** [comment]
> "Very few new code lines" for the option 1 is is a "pro" and not a "con" I guess 😉 

Indeed, was in wrong Option. It is the Option 2 that would result in few new code lines since it leverages ggplo2 a lot more.

**Comment by Yuri05** [comment]
In PK-Sim, we have only **filled** symbols, thus having "filled symbol for >=LLOQ and open symbol for <LLOQ" was a good solution.
In TLF, user can define any symbol (predefined symbols are shown [here](https://www.open-systems-pharmacology.org/TLF-Library/dev/reference/Shapes.html?q=shapes#ref-examples); additionally user could select any other symbol by its unicode value)
So options 1 or 2 could become problematic if the user selects one of the open symbols for the (non-LLOQ) values.

I would also suggest the **option 4**:  show the LLOQ values with lower transparency:

E.g. if symbols above LLOQ look like this
![grafik](https://user-images.githubusercontent.com/25061876/226345734-13461e1b-ffd6-43d4-bc15-6f0b6b13c146.png)

Then symbols below would look like this (with alpha=0.5)
![grafik](https://user-images.githubusercontent.com/25061876/226345854-ba0b199d-3773-4ce7-8dc6-8f1222433baf.png)

So for each point set alpha to: 
- 1, if y>=LLOQ 
- 0.5 (or other appropriate value) if y<LLOQ

**Comment by PavelBal** [comment]
I like @Yuri05 suggestion. In case it gets hard to see for certain colors, we could try setting some kind of a background. But we can first try with transparency only.

**Comment by Felixmil** [comment]
Great idea ! also easy to implement.
- The alpha scale will be enabled if an LLOQ is specified in the datamaping.
- A default alpha level will be set but made accessible in the `PlotConfiguration` object.

Do we need to show a legend for this ? If yes, I think it could just be two black points with the two alpha levels.

**Comment by PavelBal** [comment]
I would not show a legend for this.

**Comment by Felixmil** [comment]
Here is a preview of what I implemented:

![image](https://user-images.githubusercontent.com/34234913/226882126-f73a2bdc-95e3-4dda-8de5-220d97c4822c.png)

![image](https://user-images.githubusercontent.com/34234913/226882221-ccc98699-2c7f-48a9-b447-16d474824ca7.png)

**Comment by Felixmil** [comment]
@PavelBal @Yuri05 Any comment ? ?

**Comment by PavelBal** [comment]
Nice!
